README.md

```diff
@@ -12,20 +12,48 @@ INTRODUCTION
 ------------
 
 The Devportal module allows you to expose API reference documentation by
-uploading Swagger/OpenAPI files - as well as create, edit, import and publish
+publishing Swagger/OpenAPI files - as well as create, edit, import and publish
 conceptual documentation on your Drupal site.
 
  * For a full description of the module, visit the project page:
    https://www.drupal.org/project/devportal
-
+   
  * To submit bug reports and feature suggestions, or to track changes:
    https://www.drupal.org/project/issues/devportal
+   
+ * The customization capabilites of the Devportal module can be expanded by the 'Page builder' module. With it, the user can create highly customisable pages with reusable building blocks.
+   
+ * You can add the 'FAQ' custom module to your Devportal. Questions can be grouped by topics and these topics can be used as filters on the FAQ listing page. 
+  
+ * If you are a content editor, we recommend you to enable the 'Guides' module, which describes the custom functionalities and collects the links for the related core drupal functionalities.
```

Comment:
Firstly, content editors usually can't enable modules at all.

Secondly, it's better to write docs in passive (in order to avoid any personalization).

Thirdly, there are a bit too much `functionalities` in this sentence. ;)